src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,44 @@
 
 class Animal {
   // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+
+    Animal.alive.push(this);
+  }
+
+  static updateAliveList() {
+    Animal.alive = Animal.alive.filter(animal => animal.health > 0);
```

Comment:
don't think we need a static method. we can use this lile in bite method

Reply:
I prefer to separate this logic